src/PerformanceTests/Tests/Tools/TestEnvironment.cs

```diff
@@ -153,7 +153,8 @@ static void Clone(string src, string dst)
                 if (equalTimestamps) return;
                 File.Delete(dst);
             }
-            if (!SymbolicLink.Create(src, dst)) File.Copy(src, dst);
```

Comment:
SymbolicLink.Create never returned true on my machine so I removed it. Is there something required for symlinks to work? @ramonsmits

Reply:
Should we set the override flag to true https://msdn.microsoft.com/en-us/library/9706cfs5(v=vs.110).aspx?

Reply:
@timbussmann You have to run with elevated permissions

- https://github.com/Particular/EndToEnd/blob/master/src/PerformanceTests/readme.md#run-visual-studio-as-administrator-to-save-disk-space

Reply:
@danielmarbach The code already does a `File.Exists(dst)` so that boolean isn't required.

Reply:
ah I missed the part about required elevated permissions, given that condition, I am even more inclined to remove this code